XenAdminTests/TabsAndMenus/MainMenuGeorge.cs

```diff
@@ -128,7 +128,7 @@ public void MainMenu_XenCenterNode()
                 new ExpectedTextMenuItem("Ena&ble PVS-Accelerator...", false, false  ),
                 new ExpectedTextMenuItem("Disable P&VS-Accelerator", false, false  ),
 	            new ExpectedSeparator(),
-	            new ExpectedTextMenuItem("Inst&all " + Branding.PRODUCT_BRAND + " Tools...", false, false  ),
+	            new ExpectedTextMenuItem("Inst&all " + "[Citrix VM Tools]" + "...", false, false  ),

```

Comment:
This file is not getting rebranded during the build (*.cs files shouldn't, generally, be), so the `[Citrix VM Tools]` will never be replaced. That's why we need to use constants declared in branding.cs (which will contain the correctly branded values, as we apply the branding to that file). See also my comment here https://github.com/xenserver/xenadmin/pull/2185#issuecomment-418325028